windows-agent/cmd/ubuntu-pro-agent/agent/agent.go

```diff
@@ -212,6 +188,13 @@ func (a *App) SetArgs(args ...string) {
 	a.rootCmd.SetArgs(args)
 }
 
+// Config returns the DaemonConfig for test purposes.
+//
+//nolint:revive
+func (a App) Config() daemonConfig {
+	return a.config
+}
+
```

Comment:
If that's just for testing you should move that method to a file in the same package named `export_test.go`. This way the method is only visible in tests.